README.md

```diff
@@ -0,0 +1,60 @@
+# The Tic-Tac-Toe
+>The purpose of this project was to demonstrate our learned skills on Ruby programming. Here we developed the Tic-Tac-Toe game.
+
+<img src="screenshot/screen.png">
+
+In this project we created an executable file on main.rb that allows the users to play the game through terminal
+
+## How To Play
+
+"Tic Tac Toe" is a 2 player game which includes 9 squares each of them represented with a number. Each player will be represented with a symbol ("X" or "O") and their respective symbols will overwrite the square numbers.The first player will start the game.
+If after 9 turns neither player manages to put their respective symbols in a straight/diagonal line the game will end as a draw
+
+## Build with
+
+- Ruby
+- VSCode
+- Rubocop
+
+## Getting Started
+
+In order to play it locally follow the next steps
+
+### Prerequisites 
+
+Ruby installed localy 
+
+### Install
+
+1. Clone the repo into your machine
+    $ https://github.com/GzimAsani/Tic-Tac-Toe-Game.git
```

Comment:
it should be written like this `clone https://github.com/GzimAsani/Tic-Tac-Toe-Game.git`